source/educators/references/resources_for_students.rst

```diff
@@ -1,14 +1,14 @@
-******************************
-Resources for edx.org Learners
-******************************
+###################################
+Resources for Open edX.org Learners
```

Comment:
There is no "openedx.org" site. Please change all references to "edx.org" to simply "Open edX".

Reply:
Done!